src/components/ViewDelegateeItem.jsx

```diff
@@ -1,5 +1,5 @@
 // React + Web3 Essentials
-import { useEffect, useState } from 'react';
+import React, { useEffect, useState } from 'react';
```

Comment:
useState is already imported so 
- We don't need to import React and do React.useState, not a good approach.
- Also this imports the whole library in this file instead of what is actually required.

Reply:
agreed